baselines/splunk_risky_command_abuse_ml_detection_baseline.yml

```diff
@@ -0,0 +1,71 @@
+name: Splunk Risky Command Abuse ML Detection Baseline
+id: 273df2f7-643a-451a-8d4d-637e39eadc87
+version: 1
+date: '2022-05-27'
+author: Abhinav Mishra, Kumar Sharad and Xiao Lin, Splunk
+type: Baseline
+datamodel:
+- Splunk_Audit
+description: 'This search is to build a per user risky command abuse detection model using MLTK DensityFunction 
```

Comment:
@xlinsplunk I would update the name of the detection here, if you are going to update the search name, I would also include it as part of the how to implement.

Reply:
done